vault/expiration_test.go

```diff
@@ -2828,7 +2828,11 @@ func TestExpiration_MarkIrrevocable(t *testing.T) {
 	}
 
 	irrevocableErr := fmt.Errorf("test irrevocable error")
+
+	exp.pendingLock.Lock()
 	exp.markLeaseIrrevocable(ctx, loadedLE, irrevocableErr)
+	irrevocableLeaseCount := exp.irrevocableLeaseCount
```

Comment:
since this is a test and we aren't overly-concerned about performance, it's probably fine to declare `irrevocableLeaseCount` closer to where it's used and just lock/unlock twice

Reply:
makes sense, changed here df2931a